packages/webdriver/tests/request.test.js

```diff
@@ -30,7 +30,7 @@ describe('webdriver request', () => {
 
     describe('createOptions', () => {
         it('fails if command requires sessionId but none given', () => {
-            const req = new WebDriverRequest('POST', '/wd/hub/session/:sessionId/element')
```

Comment:
Shall we move path to constant in the top of the file to reduce possible refactoring in later on?